tests/test_collector.py

```diff
@@ -126,6 +126,15 @@ def test_collector_collect_path_symlink_follow(mock_collector) -> None:
         mock_collector.collect_symlink.assert_called()
 
 
+def test_collector_collect_glob(mock_collector) -> None:
+    with patch.object(mock_collector, "collect_file", autospec=True):
+        mock_collector.collect_glob(
+            "/foo/bar/*",
+            module_name=MOCK_MODULE_NAME,
+        )
+        mock_collector.collect_file.assert_called()
```

Comment:
Shouldn't it also check if it was called with the `module_name=MOCK_MODULE_NAME` parameter?

Reply:
Added